template/python3-flask/Dockerfile

```diff
@@ -34,13 +34,20 @@ WORKDIR /home/app/function/
 COPY function/requirements.txt	.
 RUN pip install --user -r requirements.txt
 
-WORKDIR /home/app/
-
 #install function code
 USER root
 
-COPY function   function
-RUN chown -R app:app ./
+COPY function/   .
+RUN chown -R app:app ../
+
+ARG TEST_COMMAND=tox
+RUN if [ "x$TEST_COMMAND" = "xoff" ]; then \
```

Comment:
Silly question...why does this work and disable tox? Shouldn't it be looking for a mach of `xoff`?

`faas-cli build -f echo0.yml --build-arg TEST_COMMAND='off'`

I would of expected it to still run tox, and then we'd need something like this to disable tox:

`faas-cli build -f echo0.yml --build-arg TEST_COMMAND='xoff'

Reply:
this works to disable tox because the script is check for "off" and just printing an message, if the command is not exactly off, then it runs the test command, which defaults to tox

```
if "off":
  print "disabled";
else:
  eval "test command";
```

Reply:
Why does xoff work with a value of off?

Given this:
`"x$TEST_COMMAND" = "xoff"` 

The `x` in `xoff` is being ignored somehow when the string comparison is done...right?

I'm guessing there's a bash thing going on that I don't quite understand.

Reply:
From Lucas:

"" cannot be use to set the TEST_COMMAND to empty
Also, "off" apparently doesn't work so "xoff" is required instead.

Reply:
I think I would err on the side of caution when overloading the build arg with a mode plus a command, so think about splitting out two variables.

`TEST_ON=true/false`
`TEST_COMMAND=go test -v ./...`
`TEST_COMMAND=go test -v ./pkg`

Reply:
Done